calculator/src/components/Totals.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Ingredient from './Ingredient'
+
+export default function Totals (props) {
+    
+    const recName = props.recName;
+    const carbs1 = props.carbs1;
+    const carbs2 = props.carbs2;
+    const carbsTotal = carbs1 + carbs2;
+    const sugars = props.sugars;
+    const measurement = props.measurement;
+
+    return (
```

Comment:
This just comes from my personal preference... I like to use redux to run my logic.

Once I have a few components broken out, it can feel crazy to me to pass info into props all the time from parents. So I like to attach all the logic/math/calculations from a single js file and distribute the props and functions needed directly to the component rather than through parents. Further reading https://react-redux.js.org/